sdcm/cluster_aws.py

```diff
@@ -470,10 +470,23 @@ def __str__(self):
             " | %s" % self.ipv6_ip_address if self.test_config.IP_SSH_CONNECTIONS == "ipv6" else "",
             self._dc_info_str())
 
+    def network_configuration(self):
+        network_config = self.remoter.sudo("cat /etc/netplan/50-cloud-init.yaml").stdout.strip()
```

Comment:
I would rather we pick names and mac addresses in a more cross platform way, that can fit all OSes and all backends.

Maybe something like that:
https://unix.stackexchange.com/a/445913/418687

I don't think we can count on netplan exact configuration location/name and structure.

Reply:
changed